build.sbt

```diff
@@ -571,3 +570,12 @@ lazy val update2_12 = Seq(
     }
   }
 )
+
+lazy val xlint = Seq(
+  scalacOptions += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12)) => "-Xlint:-unused,_"
```

Comment:
Ah I see. We have some evidence parameters that we don't directly use.

There's some related discussion [here](https://github.com/scala/scala/pull/5402#issuecomment-295261637). It sounds like scala doesn't offer great options for selectively ignoring these. Unless someone sees a better option, I'm fine with this change coming in as-is.